index.html

```diff
@@ -22,23 +22,160 @@
+            </article>
+
+            <article class="work-card">
+                <div class="work-card_picture">
+                    <img src="img/posts/multi-post2.png" alt="Multi-Post">
+                </div>
+                <h3 class="work-card_title">Multi-Post Stories</h3>
+                <div class="work-card_specs">
+                    <div class="specs_client">Canopy</div>
+                    <div class="dot"></div>
+                    <div class="specs_role">Back End Dev</div>
+                    <div class="dot"></div>
+                    <div class="specs_year">2015</div>
+                </div>
+                <p class="work-card_description">A daily selection of privately personalized reads;
+                     no accounts or sign-ups required.</p>
+                <ul class="work-card_techs">
+                    <li class="tag">html</li>
+                    <li class="tag">css</li>
+                    <li class="tag">javaScript</li>
+                </ul>
+                <button class="work-card_button">See Project</button>
+            </article>
+
+        </section>
+
+        <section class="about-myself generic-container">
+            <h2 class="generic_title">About Myself</h2>
+            <p>Hello I’m a software developer! I can 
+                help you build a product , feature or 
+                website Look through some of my work and 
+                experience! If you like what you see and 
+                have a project you need coded, don’t hestiate 
+                to contact me.</p>
+            <a href="https://wa.me/584142221022">Let’s connect</a>
+            <ul class="social_wrapper">
+                <li><a href="#"><img src="img/social1.svg" alt="Twitter"></a></li>
+                <li><a href="#"><img src="img/social2.svg" alt="LinkedIn"></a></li>
+                <li><a href="#"><img src="img/social3.svg" alt="Social M"></a></li>
+                <li><a href="#"><img src="img/social4.svg" alt="Github"></a></li>
+                <li><a href="#"><img src="img/social5.svg" alt="Another Social Media"></a></li>
+            </ul>
+            <button>Get my resume</button>
+            <h4 class="languages-title">Languages</h4>
+            <ul class="languages-wrapper">
+                <li>
+                    <img src="img/javaScript.png" alt="JavaScript">
+                    JavaScript
+                </li>
+                <li>
+                    <img src="img/HTML.png" alt="HTML">
+                    HTML
+                </li>
+                <li>
+                    <img src="img/CSS.png" alt="CSS">
+                    CSS
+                </li>
+            </ul>
+            <div class="about-myself_menu-item">
+                <h4>Frameworks</h4>
+                <img src="img/chevron-right.svg">
+            </div>
+            <div class="about-myself_menu-item">
+                <h4>Skills</h4>
+                <img src="img/chevron-right.svg">
+            </div>
```

Comment:
- [ ] As stated in the project requirements, kindly make sure you group the categories in the `About myself` section `Languages`, `Skills`, and `Framework` in a `ul` tag and each of them in an `li` tag 🙏

Reply:
Thanks, I misunderstood that requirement.